src/views/Main/Provider/SettingsProvider.js

```diff
@@ -0,0 +1,52 @@
+import React, { useState, useEffect } from 'react';
+
+import withEnv from '../../../config/withEnv';
+
+import { contextSettings } from './context';
+
+const { Provider } = contextSettings;
+const defaultSettings = {
+  favicon: '/favicon.png',
+  title: 'terra-visu',
+  theme: {
+    logo: '/images/terravisu-logo.png',
+    style: ['<URL>'],
```

Comment:
Default settings have no style.
```suggestion
    style: [],
```

Reply:
done